src/platformdirs/macos.py

```diff
@@ -1,9 +1,20 @@
 import os
 
 from .api import PlatformDirsABC
+from .unix import Unix
 
+XDG_SUPPORT = {
+    "user_data_dir": "XDG_DATA_HOME",
+    # "site_data_dir": "XDG_CONFIG_DIRS",  # Not supported
+    "user_config_dir": "XDG_CONFIG_HOME",
+    # "site_config_dir": "XDG_CONFIG_DIRS",  # Not supported
+    "user_cache_dir": "XDG_CACHE_HOME",
+    "user_state_dir": "XDG_STATE_HOME",
+    # "user_log_dir": "XDG_CACHE_HOME",  # Not supported b/c Console
+}
 
-class MacOS(PlatformDirsABC):
+
+class MacOS(Unix, PlatformDirsABC):
```

Comment:
Why does this need to extend from unix?

Reply:
That's from the old version, forgot to remove it.

Reply:
Please fix it then :+1: